src/styles/images/index.md.njk

```diff
@@ -8,53 +8,214 @@ layout: layout-pane.njk
+
+It takes effort to make charts, diagrams and complex images accessible. Consider whether you can give the same information well enough using a table or other text-based option.
+
+In general, you must write short alt text that explains the meaning and context of the image. You must also write any further, and more complex, information contained in the image in a longer text description.
+
+Make sure the long description (or a link or button that shows it) is visible to sighted users.
+
+Some charts and diagrams can be built in an accessible way using HTML, CSS, JavaScript or SVG. Accessible charts and diagrams must make the data within the chart available as text.
+
+## Alt text
+
+Alternative text, or alt text, is read out by screen readers and displayed if an image does not load or if images have been switched off.
+
+All images should include the alt attribute, even if it’s left empty without alt text (`alt=""`).
+
+### When to use alt text
+
+Only add alt text when you need to describe visual details you cannot practically include in the written content.
+
+There are some cases when an empty `alt=""` attribute is more appropriate. These include:
+
+- a decorative image that does not include important content
+- an icon with a text label, where the alt text would repeat the text label
+- an image used in a link, where the image is not needed to understand the link
+
+To decide if an image needs alt text, see the [Web Accessibility Initiative alt decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/).
+
+### Writing good alt text
+
+Good alt text:
 
 - tells people what information the image provides
 - describes the content and function of the image
 - is specific, meaningful and concise
 
 Use normal punctuation, like commas and full stops, so the text is easy to read and understand.
 
-Do not:
+Generally, aim for no more than 2 sentences of alt text.
+
+Do not write alt text that:
 
-- include the name of the photographer or person who created the image
-- start with ‘Image of’, ‘Graphic of’ or ‘Photo of’
-- repeat information from the page
-- include extra information not in the image
+- includes the name of the photographer or person who created the image
+- starts with ‘Image of’, ‘Graphic of’ or common information that would repeat across every image
+- repeats information from the page
+- includes extra information not in the image
 
 {{ example({group: "styles", item: "images", example: "alt-text", html: true, open: true, size: "l"}) }}
 
-Some images do not need descriptions. Use `alt=""` if the image is:
+### Research on Images
+
+In March 2023, we updated this page to help teams make decisions on when to use images in their services.
+
+In particular, we added a section on how to use images and ensure they’re as accessible as possible — giving advice to teams on how to meet or surpass relevant WCAG AA guidelines, in line with GOV.UK Design System accessibility strategy.
```

Comment:
Should it be 'in line with the GOV.UK Design System accessibility strategy'?

Reply:
Good catch!